cypress/integration/Section/ManagedListTest.feature

```diff
@@ -0,0 +1,67 @@
+Feature: Managed Lists Tests
+
+    Scenario Outline: Verify managed list is visible and has expected number of items
+        Given user is navigating to "<url>"
+        Then managed list section is visible
+        And number of items in the list is <count>
+        And each managed list description text is within reasonable length
+
+        Examples:
+            | url                  | count |
+            | /                    | 4     |
+            | /about-cancer        | 4     |
+            | /research            | 4     |
+            | /about-cancer/coping | 4     |
+
+
+    Scenario Outline: Verify managed list item's components
+        Given user is navigating to "<url>"
+        Then managed list section is visible
+        And list items at positions "<indexes>" have titles as "<title>"
+        And list items at positions "<indexes>" have links as "<itemLink>"
+        And list items at positions "<indexes>" have alt texts as "<altText>"
+        And list items at positions "<indexes>" have sources as "/sites/default/files/styles/cgov_thumbnail/public/cgov_image/thumbnail/" and files as "<file>"
```

Comment:
line 20 - 23 is really asking to be a single dynamic step :
And list items at positions "\<indexes\>" have "titles" as "\<title\>"
 And list items at positions "\<indexes\>" have "links" as "\<itemLink\>"
        And list items at positions "\<indexes\>" have "alt texts" as "\<altText\>"
        And list items at positions "\<indexes\>" have "sources" as "/sites/default/files/styles/cgov_thumbnail/public/cgov_image/thumbnail/"
          And list items at positions "\<indexes\>" have "files" as "\<altText\>"
          
          
          when you implement the code logic, use switch or if statement - so you implement it once instead of 4 times

Reply:
done